istio/README.md

```diff
@@ -24,18 +24,14 @@ If you want to monitor the Envoy proxies in Istio, configure the [Envoy integrat
 
 ### Configuration
 
-Edit the `istio.d/conf.yaml` file (in the `conf.d/` folder at the root of your [Agent's configuration directory][7]) to connect to Istio. See the [sample istio.d/conf.yaml][8] for all available configuration options.
-
 #### Metric collection
-To monitor the `istiod` deployment and `istio-proxy` in Istio `v1.5+`, use the following configuration:
-    
-```yaml
-init_config:
+The Istio integration has two key components for how it collects the prometheus formatted Istio metrics. This corresponds to the [Istio architecture][23] split between its data plane (the `istio-proxy` sidecar containers) and the control plane (the `istiod` service managing the proxies). These are both ran as `istio` Agent checks, however have different responsibilities and configuration methods.
```

Comment:
```suggestion
There are two key components involved in setting up the Istio integration to collect the Prometheus-formatted Istio metrics. Aligning with the [Istio architecture][23], there is the **data plane** (the `istio-proxy` sidecar containers) and the **control plane** (the `istiod` service managing the proxies). These are both run as `istio` Agent checks, but have different responsibilities, and they are  configured separately, as described below.
```

It's not often I recommend starting sentences with "there is", lol, but this is one of those times. I think it makes what you're saying a little clearer. Look okay to you?